transit_odp/timetables/dataframes.py

```diff
@@ -256,32 +258,121 @@ def vehicle_journeys_to_dataframe(
     return pd.DataFrame(all_vechicle_journeys)
 
 
-def operating_profile_to_df(operating_profiles):
-    serviced_orgs_refs_df = []
-    for operating_profile in operating_profiles:
-        serviced_organisation_day_type = operating_profile.get_element_or_none(
-            ["ServicedOrganisationDayType"]
+def operating_profiles_dataframe(vehicle_journeys, services):
+    operating_profile_df = []
+    for vehicle_journey in vehicle_journeys:
+        serviced_org_ref = ""
+        days_of_week = ""
+        operational = ""
+        vehicle_journey_code = vehicle_journey.get_element(["VehicleJourneyCode"]).text
+        service_ref = vehicle_journey.get_element(["ServiceRef"]).text
+        operating_profile_vehicle_journey = vehicle_journey.get_element_or_none(
+            ["OperatingProfile"]
         )
-        if serviced_organisation_day_type:
-            days_of_operation = serviced_organisation_day_type.get_element_or_none(
-                ["DaysOfOperation"]
-            )
-            days_of_non_operation = serviced_organisation_day_type.get_element_or_none(
-                ["DaysOfNonOperation"]
+        if operating_profile_vehicle_journey:
+            serviced_organisation_day_type = (
+                operating_profile_vehicle_journey.get_element_or_none(
+                    ["ServicedOrganisationDayType"]
+                )
             )
-            if days_of_operation:
-                operational = True
-                working_days = days_of_operation.get_element("WorkingDays")
-            elif days_of_non_operation:
-                operational = False
-                working_days = days_of_non_operation.get_element("WorkingDays")
-            serviced_org_ref = working_days.get_element("ServicedOrganisationRef").text
-
-            serviced_orgs_refs_df.append(
-                {"serviced_org_ref": serviced_org_ref, "operational": operational}
+            regular_day_type = operating_profile_vehicle_journey.get_element_or_none(
+                ["RegularDayType"]
             )
+            if regular_day_type:
+                days_of_week_elements = regular_day_type.get_elements_or_none(
```

Comment:
Will there more than one `DaysOfWeek` element?

Reply:
Not allowed by schema.

<img width="697" alt="Screenshot 2024-02-08 at 11 49 08" src="https://github.com/department-for-transport-BODS/bods/assets/108486680/97bd26e6-5e32-45a9-80e7-740cc9336dff">

Reply:
Can we use get_element in this case instead of get_elements?